src/model/operation/wrapoperation.js

```diff
@@ -31,7 +31,7 @@ export default class WrapOperation extends Operation {
 	 * @param {Number} howMany Offset size of wrapped range. Wrapped range will start at `position.offset` and end at
 	 * `position.offset + howMany`.
 	 * @param {module:engine/model/element~Element|module:engine/model/position~Position} elementOrPosition Element to
-	 * wrap with or position in graveyard before the element which should be used as a wrapper.
+	 * wrap or position in graveyard before the element which should be used as a wrapper.
```

Comment:
"Element to wrap" sounds like the element that will be wrapped. `this.element` is the wrapping element. So, okay, I understand that you think that the word "with" is incorrect here, but it cannot be like you wrote. Other suggestions:

* "Element to wrap in"
* "Wrapper element"
* "Wrapping element"